system/security/LdapSecurity/ldapsecurity.ipp

```diff
@@ -149,6 +149,7 @@ public:
 
     virtual void setPropertyInt(const char* name, int value){}
     virtual int getPropertyInt(const char* name){ return 0;}
+    IPropertyIterator * getPropertyIterator() const override { return nullptr;}
```

Comment:
Should be virtual ?

Reply:
I will leave this as-is. Per Tony Fishbeck:

The reason it is ok to use the override keyword even when the surrounding code doesn’t is that that is an improvement we are trying to make slowly throughout our codebase whenever we make changes to existing code.  It is a very good practice and therefore we think we should use it even when changing older code.

I don’t think you need to add the virtual keyword.  The virtual keyword used to serve as a sort of documentation even if the function was already virtual.  Now the override keyword serves that purpose as well.

Reply:
@timothyklemm the lone override is fine, for consistency's sake, if  the function is to be virtual, go ahead and label it as such, even if it's redundant.

Reply:
@timothyklemm your verbal explanation makes sense.